addons/website/views/snippets/s_image_text_box.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_text_box" name="Image Text Box">
+    <section class="s_image_text_box pt80 pb80">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="10" style="column-gap: 24px;">
+                <div class="o_grid_item o_grid_item_image g-col-lg-6 g-height-10 col-lg-6" style="grid-area: 1 / 1 / 11 / 7; z-index: 1; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px; order:2;">
+                    <img src="/web/image/website.s_text_cover_default_image" class="img img-fluid rounded order-lg-0" alt=""/>
+                </div>
+                <div class="o_grid_item o_cc o_cc2 g-col-lg-6 g-height-10 col-lg-6 justify-content-center rounded order-lg-0" style="grid-area: 1 / 7 / 11 / 13; z-index: 2; --grid-item-padding-y: 48px; --grid-item-padding-x: 48px; order:1;">
+                    <h3>Learn about our offerings</h3>
+                    <p>Write one or two paragraphs describing your product or services. To be successful your content needs to be useful to your readers.</p>
+                    <p>Start with the customer – find out what they want and give it to them.</p>
+                    <p class="mb-0"><a href="#" class="btn btn-primary">Learn more</a></p>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="s_image_text_box_options" inherit_id="website.snippet_options">
+    <!--  Box Vertical Alignment -->
+    <xpath expr="//div[@data-js='WebsiteAnimate']" position="after">
+        <div data-selector=".s_image_text_box .o_grid_item" data-exclude=".o_grid_item_image">
+            <we-button-group string="Vert. Alignment" title="Vertical Alignment">
```

Comment:
If you change the `layout` option to `cols`. This option no longer works.
So we have to hide it for `cols` using `data-dependencies`.
But do we really need this option ? Shouldn't it be a generic option we might add later ?
Moreover, it is already possible to set the vertical alignement by using the column padding option.

Reply:
I reset to a padding solution to align the text. I think we really need that option indeed 👍 

Thanks for the review !